extensions/ql-vscode/src/databaseFetcher.ts

```diff
@@ -339,7 +342,7 @@ function looksLikeLgtmUrl(lgtmUrl: string | undefined): lgtmUrl is string {
     }
 
     const paths = uri.path.split("/").filter((segment) => segment);
-    return paths.length === 4 && paths[0] === "projects" && paths[1] === "g";
+    return paths.length >= 4 && paths[0] === "projects";
```

Comment:
Maybe just leave a comment in here explaining the URL munging going on --- basically a copy-paste of what @aibaars told us would be good. Otherwise this whole PR lgtm.

Reply:
Sure